### DISCUSSION TITLE
After installing 1.0.0-beta14, I get an "anchorFormat 2 not supported" exception when using some fonts with IImageProcessingContext.DrawText()

### DISCUSSION THREAD

**Discussion by rambouillet** [SixLabors/ImageSharp.Drawing#210] [Q&A]
Before updating to 1.0.0-beta14, I was able to load and use any font family with `IImageProcessingContext.DrawText(String, Font, Color, PointF)`. Now, with a lot of the fonts I used to use, I'm getting the following exception:

```
NotSupportedException: anchorFormat 2 not supported. Should be '1'.
SixLabors.Fonts.Tables.AdvancedTypographic.GPos.AnchorTable.Load(BigEndianBinaryReader reader, long offset)
```

Before I post all my code and file a bug report, is this exception intended? Like I said, all my fonts were working before the update.

Note: with some fonts I don't get this exception; it only happens with some fonts.

**Comment by tocsoft** [comment]
SixLabors/Fonts#237 and SixLabors/Fonts#238 are the pre-existing issue and fix. The fix for fonts is currently in our [MyGet feed](https://www.myget.org/feed/sixlabors/package/nuget/SixLabors.Fonts) we are not sure when we will be able to the fix out to nuget are we are currently work through some large/potentially breaking changes.
